### DISCUSSION TITLE
SSL

### DISCUSSION THREAD

**Discussion by rodneyaz** [mrlt8/docker-wyze-bridge#739] [General]
Is it possible to add SSL cert to the docker-wyze-bridge installation so that all the links use https?  I have it installed in home assistant as an add-on.  While my home assistant uses port 443, I have mixed content because the camera links are all http.  It would be nice to be able to force end to end encryption, but I don't see a way to do that while the camera links are http.  Does that sound correct, or is there a way around this I'm not seeing?

**Comment by mrlt8** [comment]
The LLHLS option will reuse the SSL certificate from HAS, so we could probably extend SSL support to the other parts of the webUI and WebRTC.

**Comment by dgiaffe** [comment]
I would love to see this as well. Trying to get the WebRTC feeds on my lovelace dashboard, but whenever I use Nabucasa to remote in, I can't see the feeds from my dashboard. I did just setup my home assistant with https natively thinking (based on my basic understanding of the message) it would help, and then I stumbled upon the post here when it didn't help. At least it made my local install of Home Assistant more secure in the process haha. 

Perhaps there is another way to get the feed on the dashboard if that isn't by extending the SSL support? Loving this add-on by the way though!

**Comment by rodneyaz** [comment]
I ended up going back to http://homeassistant.local:8123 for my local.  I
setup nginx proxy manager in home assistant as an addon.  I use cloudflare
for DNS, so I setup full encryption between the cloudflare proxy and my
router using a letsencrypt cert that's managed within nginx.  My camera
entity points to the local home assistant ip and this allows me to see the
cameras inside and outside of the network.  This allowed me to close 8123
on the port forwarding and everything runs through port 443 now.
